### ISSUE TITLE
Add `Mooncake` to `Lux` benchmarking suite 

### ISSUE THREAD

**Issue by yebai** [chalk-lab/Mooncake.jl#405]
For more comprehensive benchmarking, let's add Mooncake to the Lux's benchmarking set: 

- results: https://lux.csail.mit.edu/benchmarks/
- code: https://github.com/LuxDL/Lux.jl/tree/main/benchmarks

Cc @avik-pal, who might be able to provide pointers on what is needed. 

(this issue is copied from https://github.com/compintell/Mooncake.jl/issues/361#issuecomment-2503395688)

**Comment by avik-pal** [comment]
Yes, let's do this. I need to cleanup the benchmarks on Lux end (the benchmarks are somewhat broken since the monorepo setup). I will update here once that is done

**Comment by RomeoV** [comment]
Why is this not planned anymore? I think it would be great to have this.

**Comment by yebai** [comment]
I think this is better done on the Lux side, which has the infrastructure.

**Comment by RomeoV** [comment]
@avik-pal Is something like this planned on the Lux side?

**Comment by avik-pal** [comment]
> @avik-pal Is something like this planned on the Lux side?

Kind of. We already have the code infra, but unfortunately I lack reliable GPU benchmarking CIs on LuxDL side.

**Comment by yebai** [comment]
> Kind of. We already have the code infra, but unfortunately I lack reliable GPU benchmarking CIs on LuxDL side.

@avik-pal Would RTX Pro 6000 be good enough? I am considering getting one for Turing.jl + Mooncake.jl + Lux.jl.

**Comment by avik-pal** [comment]
Yeah that would be great.
